helm/rbac-operator/templates/configmap.yaml

```diff
@@ -40,3 +40,12 @@ data:
         - name: {{ . }}
         {{- end }}
         {{- end }}
+        readAllCustomerGroups:
+        {{- if .Values.oidc.customer.read_all_group }}
+        - name: {{ .Values.oidc.customer.read_all_group }}
+        {{- end }}
```

Comment:
Is there a reason to support the singular name here? I'd be inclined to only support the plural. That's also the plan for write_all, it just still has users and it's a breaking change to remove it.

Reply:
Well, this wasn't there so I don't think anybody is actually using it. I can remove it.